### ISSUE TITLE
Change bios' characters limit to 500 by default

### ISSUE THREAD

**Issue by Sylvhem** [florence-social/mastodon-fork#42]
One of the main selling points of Mastodon always have been its 500 characters limit. Despite that, the number of characters one can use in their biography is only of 340, which is both short and a strange choice.
I would suggest to change the default limit of the number of characters that can be used in bios to 500.

**Comment by renatolond** [comment]
I did a similar change in mastodonte, and I was thinking that we could follow the change on #32 to make it configurable by the interface and all that.
As far as I know (I looked over different app issues at the time) there's no equivalent config key to inform that an instance supports a different bio size as it's the case with toots. The apps try with a bigger size and show the error message if given.
